functions/openhab.bash

```diff
@@ -30,7 +29,6 @@ delayed_rules() {
 
   targetDir="/etc/systemd/system/openhab2.service.d"
 
-  create_sys_dependencies
```

Comment:
Leave in, otherwise disabling won't work properly in the user menu.

Reply:
I don't like this nesting. Added calling this to menu.bash on user selecting delayed_rules()